### ISSUE TITLE
Is websocket protocol level ping/pong supported?

### ISSUE THREAD

**Issue by vspinu** [ztellman/aleph#322]
I am having a strange issue with pusher feeds. It keeps sending 

```json
{"event":"pusher:error","data":{"code":4201,"message":"Pong reply not received"}}
```
with no preliminary `pusher:pong` event. The [documentation states](https://pusher.com/docs/pusher_protocol#ping-pong):

> 
> If the WebSocket connection supports ping & pong (i.e. advertises itself as draft 01 or above), Pusher will send ping messages to the client in order to verify that it is active.
> 
> In protocol versions 5 and above, when using an old version of the WebSocket protocol, Pusher will send pusher:ping event (see events) to the client. The client should respond with a pusher:pong event.
> 

From this I could only infer that aleph's web socket client advertises itself as 01 or above but doesn't support protocol level ping/pong. Is this correct? How should I go to fix this? 

Thank you.

**Comment by vspinu** [comment]
Without having too much clue about the internals of websockets, a quick look at the implementation of the handler reveals that only pong frame is [treated](https://github.com/ztellman/aleph/blob/master/src/aleph/http/client.clj#L453) explicitly. But, judging from [PongWebSocketFrame examples](http://www.programcreek.com/java-api-examples/index.php?api=io.netty.handler.codec.http.websocketx.PongWebSocketFrame) there is a `PingWebSocketFrameClass` that should be taken care of as well.

**Comment by shilder** [comment]
https://tools.ietf.org/html/rfc6455#section-5.5.2

#351

**Comment by kachayev** [comment]
Closed with #364.
